test/parallel_api/iterator/zip_iterator_equal.pass.cpp

```diff
@@ -0,0 +1,179 @@
+// -*- C++ -*-
+//===-- zip_iterator_equal.pass.cpp ---------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+#include "support/utils.h"
+
+#include _PSTL_TEST_HEADER(execution)
+#include _PSTL_TEST_HEADER(algorithm)
+
+#if !defined(_PSTL_TEST_EQUAL) && \
+    !defined(_PSTL_TEST_FOR_EACH_STRUCTURED_BINDING)
+#define _PSTL_TEST_EQUAL
+#define _PSTL_TEST_EQUAL_STRUCTURED_BINDING
+#endif
+
+using namespace TestUtils;
+
+#if TEST_DPCPP_BACKEND_PRESENT
```

Comment:
I propose write all includes in the begin of the file: it's better for reading.

Reply:
I have moved the includes to the beginning of each file.